samples/nextjs/src/components/graphql/GraphQL-ConnectedDemo.dynamic.tsx

```diff
@@ -31,6 +32,10 @@ const GraphQLConnectedDemo = (props: StyleguideComponentProps): JSX.Element => {
     ? useComponentProps<GraphQLConnectedDemoData>(props.rendering.uid)
     : undefined;
 
+  useEffect(() => {
+    resetEditorChromes();
```

Comment:
Should we only call this if `isEditorActive()`? Even though the code works as-is, I think adding the check makes it more clear that this is an editing-only thing.